types/index.d.ts

```diff
@@ -0,0 +1,66 @@
+declare namespace ShopifyToken {
+    export interface ShopifyTokenOptions {
+        // The redirect URL for the Oauth2 flow
+        redirectUri: string;
+        // The Shared Secret for the app
+        sharedSecret: string;
+        // The API Key for the app
+        apiKey: string;
+        // The list of scopes
+        scopes?: string | string[];
+        // The request timeout
+        timeout?: number;
+    }
+}
+
+interface ShopifyToken {
```

Comment:
Can we use `declare class` instead of `interface`?

Reply:
you can but I was having issues exporting it that case

Reply:
Only asked because that's what we use here https://github.com/primus/eventemitter3/blob/master/index.d.ts

Reply:
I believe the difference is that this also has to export a second interface but I can take another look